src/Optimizely/ProjectConfig.php

```diff
@@ -647,6 +648,12 @@ public function setForcedVariation($experimentKey, $userId, $variationKey)
             return false;
         }
 
+        // check for empty string Variation key
+        if (!is_null($variationKey) && !Validator::validateNonEmptyString($variationKey)) {
+            $this->_logger->log(Logger::DEBUG, 'Variation key is invalid');
```

Comment:
For Error message use INVALID_FORMAT in Errors.php class.

Reply:
Should be Logger::ERROR. We log all invalid inputs with level Error in Optimizely class